R/inputs.R

```diff
@@ -112,7 +112,7 @@ updateComboBox.shinyInput <- shiny.react::updateReactInput
 
 #' @rdname DatePicker
 #' @export
-DatePicker.shinyInput <- input("DatePicker", NULL)
+DatePicker.shinyInput <- input("DatePicker", shiny.react::JS("new Date()"))
```

Comment:
Let's use R function to achieve the same effect:
```suggestion
DatePicker.shinyInput <- input("DatePicker", Sys.Date())
```